### DISCUSSION TITLE
Inventree + Nginx Proxy Manager

### DISCUSSION THREAD

**Discussion by E-Hain** [inventree/InvenTree#5459] [Q&A] [answered]
So I'm setting up Inventree in a docker container. The documentation says to use Nginx Proxy. However, I already use Nginx Proxy Manager for my other docker containers. Is it possible to replace the recommended Nginx Proxy and only use NPM instead? If it's possible, are there any available examples of implementation?

Thanks!

**Comment by matmair** [comment]
@E-Hain should be possible, I am running nearly all my instances behind caddy for example. I do not know of public examples of NPM settings. Reading the default nginx template should give good hints, we are not using exotic commands.

**Reply by E-Hain** [reply]
@matmair thought it would be possible but so far I haven't had any luck

**Comment by wolflu05** [comment, marked as answer]
Hello @E-Hain ,

I don't think that will be easy. You would need to serve the static files via NPM which is not the way NPM is intended for. To not mix different services and their volumes I would suggest using the internal nginx docker container like documented and then setup a proxy host to the internal inventree nginx proxy container in NPM.
